packages/ui/src/components/TaskCard/TaskCardItem.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from "react";
+
+export interface TaskCardItemProps {
+  children?: ReactNode;
+}
+
+const TaskCardItem = ({ children }: TaskCardItemProps) => {
+  return (
+    <div
+      data-h2-padding="base(x1) p-tablet(x1 x1.5)"
+      data-h2-border-bottom="base:selectors[:not(:last-child)](1px solid gray.lighter)"
+    >
+      {children}
+    </div>
+  );
+};
```

Comment:
I don't think this is what Josh meant, cause this component is only a card. The heading padding should match the interior padding, so you only needed to add the padding to the `<div>{children}</div>`. However, looking at the screenshot of the #10785 Dashboard view, this would keep things consistent so I like it :+1: !

Sorry to be nit picky, but could we make this and TaskCard into a compound component instead? Similar to the  [DescriptionList](https://github.com/GCTC-NTGC/gc-digital-talent/blob/11075-task-card-component/packages/ui/src/components/DescriptionList/DescriptionList.tsx#L1-L115). You just need to add this into the TaskCard, and rename this to Item, and rename the TaskCard to Root. Then we can use it similar to our other components: `<TaskCard.Root /><TaskCard.Item>{...}</TaskCard.Item></TaskCard.Root>`

Reply:
I like consistency.  If we end up using this card for non-list content we might have to undo this though.
[refactor to compound component](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11161/commits/7b4c5222782220a76328aba072e307297ad58d2c)